staging/src/k8s.io/legacy-cloud-providers/azure/azure_controller_common.go

```diff
@@ -58,17 +57,16 @@ var defaultBackOff = kwait.Backoff{
 	Jitter:   0.0,
 }
 
-// acquire lock to attach/detach disk in one node
-var diskOpMutex = keymutex.NewHashed(0)
```

Comment:
@aramase have you tried give a big number for NewHashed func? that could be simpler fix.

Reply:
I was wrong, your fix is based on string entry, it's a right fix.